controllers/cloud.redhat.com/providers/metrics/none.go

```diff
@@ -0,0 +1,24 @@
+package metrics
+
+import (
+	crd "cloud.redhat.com/clowder/v2/apis/cloud.redhat.com/v1alpha1"
+	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/config"
+	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/providers"
+)
+
+type noneMetricsProvider struct {
+	providers.Provider
+}
+
+func NewNoneMetricsProvider(p *providers.Provider) (providers.ClowderProvider, error) {
+	return &noneMetricsProvider{Provider: *p}, nil
+}
+
+func (m *noneMetricsProvider) Provide(app *crd.ClowdApp, c *config.AppConfig) error {
+
+	if err := createMetricsOnDeployments(m.Cache, m.Env, app, c); err != nil {
```

Comment:
Is there a reason we create metrics in the `none` provider? The other `none` providers seem to return either empty structs or nil in their `Provide()` methods.

Reply:
It's a good question, it's a base level of functionality that we always want, the none refers to things like service monitors etc that never get created in None mode.